### PR TITLE
feat: forward kube_hpa_status_condition metric

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

 forward kube_hpa_status_condition metric
 
 solves #1605 

---

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Review by pmalek-sumo — APPROVED** [review]

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -1500,13 +1500,14 @@ kube-prometheus-stack:
...
+              regex: kube-state-metrics;(?:kube_statefulset_status_observed_generation|kube_statefulset_status_replicas|kube_statefulset_replicas|kube_statefulset_metadata_generation|kube_daemonset_status_current_number_scheduled|kube_daemonset_status_desired_number_scheduled|kube_daemonset_status_number_misscheduled|kube_daemonset_status_number_unavailable|kube_deployment_spec_replicas|kube_deployment_status_replicas_available|kube_deployment_status_replicas_unavailable|kube_node_info|kube_node_status_allocatable|kube_node_status_capacity|kube_node_status_condition|kube_hpa_spec_max_replicas|kube_hpa_spec_min_replicas|kube_hpa_status_(condition|(current|desired)_replicas)
```
Comment (Bryce-L) [inline deploy/helm/sumologic/values.yaml:1510]: It looks like you're missing a closing ')' at the end of the line.  There are 3 '(' but only 2 ')'.  I tried applying these changes locally and prometheus-sumologic-kube-prometheus-prometheus-0 started complaining with the following error:

parsing YAML file /etc/prometheus/config_out/prometheus.env.yaml: error parsing regexp: missing closing ): `^(?:kube-state-metrics;(?:kube_statefulset_status_observed_generation|kube_statefulset_status_replicas|kube_statefulset_replicas|kube_statefulset_metadata_generation|kube_daemonset_status_current_number_scheduled|kube_daemonset_status_desired_number_scheduled|kube_daemonset_status_number_misscheduled|kube_daemonset_status_number_unavailable|kube_deployment_spec_replicas|kube_deployment_status_replicas_available|kube_deployment_status_replicas_unavailable|kube_node_info|kube_node_status_allocatable|kube_node_status_capacity|kube_node_status_condition|kube_hpa_spec_max_replicas|kube_hpa_spec_min_replicas|kube_hpa_status_(condition|(current|desired)_replicas))$`

Comment (Bryce-L) [reply deploy/helm/sumologic/values.yaml:1510]: Confirmed, I got it working by adding a ')' to the end of the line.

```suggestion
              regex: kube-state-metrics;(?:kube_statefulset_status_observed_generation|kube_statefulset_status_replicas|kube_statefulset_replicas|kube_statefulset_metadata_generation|kube_daemonset_status_current_number_scheduled|kube_daemonset_status_desired_number_scheduled|kube_daemonset_status_number_misscheduled|kube_daemonset_status_number_unavailable|kube_deployment_spec_replicas|kube_deployment_status_replicas_available|kube_deployment_status_replicas_unavailable|kube_node_info|kube_node_status_allocatable|kube_node_status_capacity|kube_node_status_condition|kube_hpa_spec_max_replicas|kube_hpa_spec_min_replicas|kube_hpa_status_(condition|(current|desired)_replicas))
```

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml:1510]: Thanks. Fixing in #1635

**Comment by perk-sumo** [comment]
I moved this int `v2.2` and removed the `breaking-change` label as adding this particular metric is not breaking anything.

**Pull request merged**